mongo/bounties/init.js

```diff
@@ -1,4 +1,15 @@
-conn = new Mongo();
+console.log('Attempting to connect to mongo...');
+
+// Replaces `localhost` if running in a Docker container on network `mongo`
+// for more info, see https://docs.docker.com/network/
+conn = new Mongo('mongo:27017');
+
+console.log('Connected to Mongo, fetching the DB...');
+
 db = conn.getDB("bountyboard");
-db.bounties.createIndex({ title: 'text' });
-printjson( db.bounties.createIndex({ title: 'text' }) );
\ No newline at end of file
+
+const idx = db.bounties.createIndex({
+    "$**": 'text'
```

Comment:
This is the one-liner that is the key bit of functionality changed. Obviously approving this PR doesn't implement the change (needs to happen at the DB level), but the integration tests will make it happen